tests/test_modelcard.py

```diff
@@ -0,0 +1,199 @@
+import logging
```

Comment:
Please align with other tests in the library by putting all test in a class that inherits from `unittest.TestCase`, and using `self.assertXXX` instead of `assert` statements.

Reply:
:( It looks so much cleaner this way though, imo. But okay 😛

Reply:
Cleanliness is a subjective point of view while conventions are not :wink: